src/codegen/codegen.utils.ts

```diff
@@ -109,3 +109,15 @@ export function mergeRedirects(recording: ProxyData[]) {
 
   return result
 }
+
+export const removeWebsocketRequests = (recording: ProxyData[]) => {
+  return recording.filter((data) => {
+    return getUpgradeHeader(data.request.headers) !== 'websocket'
+  })
+}
+
+export const recordingCleanup = (recording: ProxyData[]) => {
```

Comment:
Maybe `cleanupRecording`? Just so the naming is consistent across functions and the function name describes an action